app/controllers/posts_controller.rb

```diff
@@ -1,13 +1,17 @@
 class PostsController < ApplicationController
   def index
+    @comments = Comment.all
     @user = User.find(params[:user_id])
-    @posts = @user.posts
+    @posts = @user.posts.order(created_at: :desc)
```

Comment:
- [ ] Please make sure that the app solves the N+1 problem when fetching all posts and their comments for a user as requested in the project requirements.
  - Example in the posts#index actions you should use `@posts = @user.posts.includes(:comments)` instead of `@user.posts.order(created_at: :desc)`

Reply:
Resolved

Reply:
This change has been done well